config/checkstyle-checks.xml

```diff
@@ -558,7 +558,13 @@
     <module name="OneStatementPerLine"/>
     <module name="OverloadMethodsDeclarationOrder"/>
     <module name="PackageDeclaration"/>
+    <module name="ParameterAlignment">
+      <property name="severity" value="ignore"/>
+    </module>
     <module name="ParameterAssignment"/>
+    <module name="ParameterPlacement">
```

Comment:
Placing two Checks in one PR will not work well.

ParameterAlignment
ParameterPlacement

Should be separated from each other and they should make sense independently.
If not planned to be independent, they should be combined.